### PR TITLE
Add generated Go code.

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: gen-go/statsproto/doc.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package statsproto contains generated Protobuf code for OpenCensus stats.
+package statsproto
```
Comment (sebright) [inline gen-go/statsproto/doc.go]: stats.proto hasn't been updated for the new design in https://github.com/census-instrumentation/opencensus-specs yet.

Comment (ramonza) [reply gen-go/statsproto/doc.go]: Not sure what the new design is but let's fix that separately.

**Comment by tsloughter** [comment]
I wouldn't this just live in the Go repo?

**Comment by ramonza** [comment]
@tsloughter  https://github.com/census-instrumentation/opencensus-go/pull/406
not exactly sure why (I'm sure there's a good reason) but there's already code in this repo to support Java so I don't feel too bad adding Go as well.

File: gen-go/statsproto/doc.go (hunk 1 above)
Comment (sebright) [reply gen-go/statsproto/doc.go]: Some of the data types have completely changed, such as MeasurementDescriptor, DistributionAggregation, and IntervalAggregation.  I agree that it should be updated separately from this PR, though.

**Comment by tsloughter** [comment]
Ah, ok. Instead of a submodule we just put it in the readme for the Erlang library, hehe https://github.com/census-instrumentation/opencensus-erlang#updating-opencensus-standard-protobuf-encoder-and-decoder

**Pull request merged**